app/controllers/api/v1/appointments_controller.rb

```diff
@@ -0,0 +1,50 @@
+class Api::V1::AppointmentsController < ApplicationController
+  before_action :authenticate_request
+
+  def index
+    @appointments = Appointment.all
+    render json: @appointments
+  end
+
+  def show
+    @appointment = Appointment.find(params[:id])
+    render json: @appointment
+    # @appointment = Appointment.joins(:user, :trainer).find(params[:id])
+    # render json: @appointment, include: [:user, :trainer]
```

Comment:
- [ ] Kindly, remove this old commented code in order to improve your repo's cleanliness. 👍